deepchecks/vision/checks/distribution/image_property_drift.py

```diff
@@ -95,7 +92,7 @@ def update(
         else:
             raise RuntimeError(
                 'Internal Error! Part of code that must '
-                'be unreacheable was reached.'
+                'be unreachable was reached.'
```

Comment:
A more descriptive error would be "Should not reach here! unknown dataset_kind: {dataset_kind}"